File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length()) return false;
        
        int[] temp = new int[26];
        
        for(char c : s.toCharArray()){
            temp[c - 'a']++;
        }
        for(char c : t.toCharArray()){
            temp[c - 'a']--;
        }
        
        for(int i : temp){
            if(i != 0) return false;
        }
        
        return true;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False
        
        ca = [0]*26
        for i in range(len(s)):
            ca[ord(s[i]) - ord('a')] += 1
            ca[ord(t[i]) - ord('a')] -= 1
        
        for i in ca:
            if i != 0:
                return False
        return True